localisation/00_zuwc_united_localisation.csv

```diff
@@ -2655,7 +2655,7 @@ EVTDESC_WCUND_1301;Your past gods turned their back on you after you were strick
 EVTOPTA_WCUND_1301;I shall bear this burden, but I will not forsake my freedom.;;Ich werde diese Last tragen, aber ich werde meine Freiheit nicht aufgeben.;;;;;;;;;;;x
 EVTOPTB_WCUND_1301;It is a price I am willing to pay to bask in the power of the Lich King.;;Es ist ein Preis, den ich bereit bin zu zahlen, um mich in der Macht des Lichk�nigs zu sonnen.;;;;;;;;;;;x
 EVTOPTC_WCUND_1301;I shall bind my soul to the Blighted Lands, so I can draw my power from there.;;Ich werde meine Seele an die verw�steten Lande binden, damit ich meine Kraft von dort beziehen kann.;;;;;;;;;;;x
-I_WILL_NOT_BETRAY_RELIGION;I will not betray [Root.TrueReligion.GetHighGodName]!;;Ich werde [Root.TrueReligion.GetHighGodName] nicht verraten!;;;;;;;;;;;x
+I_WILL_NOT_BETRAY_RELIGION;I hope [Root.TrueReligion.GetHighGodName] will forgive me.;;Vergib mir, [Root.TrueReligion.GetHighGodName]!;;;;;;;;;;;x
```

Comment:
Hoffentlich wird mir [Root.TrueReligion.GetHighGodName] vergeben.

Reply:
What, how did it.. I pushed the changes with the.. German.. What?! I'm confused, I'll fix it but it was fixed locally. >->

Reply:
had that problem once as well, thought that I had pushed something, switched branch, and afterwards it was gone ...